frontend/src/lib/components/Cards/InsightCard/InsightDetails.tsx

```diff
@@ -158,8 +158,8 @@ function SeriesDisplay({
     const hasBreakdown =
         isInsightQueryWithBreakdown(query) &&
         query.breakdownFilter != null &&
-        query.breakdownFilter.breakdown_type != null &&
-        query.breakdownFilter.breakdown != null
+        ((query.breakdownFilter.breakdown_type != null && query.breakdownFilter.breakdown != null) ||
```

Comment:
Might be good to extract a helper function for `isBreakdown`.

Reply:
Nice suggestion. I created a type guard for that.